app/src/main/java/com/geekskool/dhobi/Adapters/StudentAdapter.java

```diff
@@ -0,0 +1,56 @@
+package com.geekskool.dhobi.Adapters;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.geekskool.dhobi.Models.Student;
+import com.geekskool.dhobi.R;
+
+import java.util.List;
+
+/**
+ * Created by manisharana on 19/2/17.
+ */
+public class StudentAdapter extends RecyclerView.Adapter<StudentAdapter.StudentViewHolder> {
+    private List<Student> studentList;
```

Comment:
As a part of less aggressive "object calisthenics", try to wrap integral types and collections wherever you can. Rather than `string name;` it provides a clear entry point to name-related behaviours to have a Name class: `Name name;`

For collections, always prefer this: `private Students` over `List<Student>`, where `Students` is a custom collection wrapping (NOT inheriting from) Java's List type.